Beginner_Projects/Tune_Tracker/Tune_tracker.py

```
@@ -0,0 +1,169 @@
+        'Cheap Thrills': {'artist': 'Sia', 'genre': 'Pop', 'mood': 'party'},
+        'Radioactive': {'artist': 'Imagine Dragons', 'genre': 'Rock', 'mood': 'intense'},
+        'Let Me Love You': {'artist': 'DJ Snake', 'genre': 'Electronic', 'mood': 'romantic'},
+        'Heat Waves': {'artist': 'Glass Animals', 'genre': 'Indie', 'mood': 'nostalgic'},
+        'Peaches': {'artist': 'Justin Bieber', 'genre': 'Pop', 'mood': 'chill'},
+        'Night Changes': {'artist': 'One Direction', 'genre': 'Pop Rock', 'mood': 'soft'},
+        'Halo': {'artist': 'Beyoncé', 'genre': 'R&B', 'mood': 'inspirational'},
+        'Levitating': {'artist': 'Dua Lipa', 'genre': 'Disco Pop', 'mood': 'happy'},
+        'Faded': {'artist': 'Alan Walker', 'genre': 'Electronic', 'mood': 'melancholic'},
+        'Closer': {'artist': 'The Chainsmokers', 'genre': 'EDM', 'mood': 'romantic'},
+        'Stressed Out': {'artist': 'Twenty One Pilots', 'genre': 'Alternative', 'mood': 'relatable'},
+        'Heat Waves': {'artist': 'Glass Animals', 'genre': 'Indie Pop', 'mood': 'nostalgic'},
```

Ruff (0.11.9)

Dictionary key literal `'Heat Waves'` repeated

(F601)